src/flows/recommendation_api/legacy_topics_flow.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
+from prefect import Flow, Parameter, unmapped, task
+from prefect.executors import LocalDaskExecutor
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from common_tasks.corpus_candidate_set import (
+    validate_candidate_items,
+)
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+from api_clients.sqs import put_results, RecommendationCandidate, NewTabFeedID
+
+FLOW_NAME = get_flow_name(__file__)
+
+# Export approved candidate items by language and recency
+EXPORT_CORPUS_ITEMS_SQL = """
+SELECT 
+    a.resolved_id as "ID", 
+    c.top_domain_name as "PUBLISHER"
+FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS" as a
+JOIN "ANALYTICS"."DBT".content as c ON c.content_id = a.content_id
+WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD('day', -90, current_timestamp())
+AND a.TOPIC = %(CORPUS_TOPIC_ID)s
+AND a.LANGUAGE = 'EN'
+"""
+
+GET_TOPICS_SQL = """
+select 
+    CURATEDCANDIDATEID as "CURATED_CORPUS_CANDIDATE_SET_ID",
```

Comment:
This is not a corpus candidate set. Let’s call this column something like “legacy_curated_topic_candidate_set_id”?

Reply:
> This is not a corpus candidate set. Let’s call this column something like “legacy_curated_topic_candidate_set_id”?

We will have to do this in a separate Snowflake dbt PR. This is a Prefect PR that ingests the candidate set to SQS